src/main/java/org/folio/service/PrefixService.java

```diff
@@ -75,4 +75,19 @@ private CompletableFuture<Void> checkPrefixNotUsed(Prefix prefix, RequestContext
         }
       });
   }
+
+  public CompletableFuture<Void> validatePrefixAvailability(String prefixName, RequestContext requestContext) {
+    if(Objects.nonNull(prefixName)) {
```

Comment:
Please change to StringUtils.isNotEmpty(), because we should not check prefix availability for empty strings

Reply:
fixed